src/model/systemmessage.h

```diff
@@ -103,6 +105,14 @@ struct SystemMessage
         case SystemMessageType::messageSendFailed:
             translated = QObject::tr("Message failed to send");
             break;
+        case SystemMessageType::selfJoinedGroup:
+            translated = QObject::tr("You have joined the group");
+            numArgs = 0;
```

Comment:
All other cases with 0 args leave out the numArgs = 0, but I'm fine with leaving this in. It's clearer to always set it with the message imo. :x

Reply:
Good point, just scrapped the whole numArgs concept and substitute in-place now.